WindowsServerDocs/manage/windows-admin-center/azure/azure-extended-network.md

```diff
@@ -0,0 +1,185 @@
+
+> [!NOTE] 
+> Windows Admin Center requires an additional machine to use as the Windows Admin Center gateway.  Do not use either of the two Virtual Machines that you created above as the Windows Admin Center gateway.  You may need to create an additional machine for this purpose.
+
+## Deploy
+
+Deployment is driven through Windows Admin Center.  
+
+### Install and configure Windows Admin Center
+
+1. [Download and install Windows Admin Center](../understand/windows-admin-center.md) onto any computer capable of running Windows Admin Center other than the two virtual appliances.  
+2. Add Azure Extended Network from the feed.  To do this, select **Settings**:
+![Image](../media/azure-extended-network/settings.png)
+Then select **Extensions**:
+![Image](../media/azure-extended-network/installed-extensions.png)
+In the Available extensions list, select **Azure Extended Network**, then click **Install**.
+After a few seconds you should see a message indicating a successful installation:
+![Image](../media/azure-extended-network/installation-success.png)
+3. Connect your Windows Admin Center deployment to Azure, if you haven’t already done so.  If you skip this step now, you will be prompted to do so later during the configuration process.
+4. Add a connection to the on-premises virtual appliance.  Return to the connection page and Select **Add** and enter the connection information (machine name and credentials) for the On-premises virtual appliance. Click on the server to connect and you should see Azure Extended Network in the list of tools for this server:
+![Image](../media/azure-extended-network/azure-extended-network.png)
+Click on **Azure Extended Network** to begin.  The first time you will be presented with an overview and a setup button:  
+![Image](../media/azure-extended-network/azure-extended-network-intro.png)
+
+### Deploy Azure Extended network
+
+1. Click **Set up** to begin the configuration.
+2. Click **Next** to proceed past the Overview.
+3. On the Upload Package panel, you will need to download the Azure Extended Network agent package and upload it to the virtual appliance.  Follow the instructions on the panel.  
+
+> [!IMPORTANT] Scroll down if necessary and click **Upload** before you click **Next: Extended-Network Setup**.
+
+4. Select the Subnet CIDR of the On-Premises network that you want to extend.  The list of subnets is read in from the virtual appliance.  If you have not connected the virtual appliance to the correct set of subnets, you will not see the desired subnet CIDR in this list.
+Click Next after selecting the Subnet CIDR.
+5. Select the Subscription, Resource Group and Virtual Network that you are extending into:
+![Image](../media/azure-extended-network/azure-network.png) 
+The Azure Location and Azure Subnet will be selected automatically.  Click Next: Extended-Network Gateway Setup to proceed.
```

Comment:
```suggestion
The region (Azure location) and subnet are selected automatically. Select Next: Extended-Network Gateway Setup to proceed.
```

Reply:
I think the UI text could be improved here - I think we use [Azure region](https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=28019) instead of "Azure location", but we haven't been super consistent on that... Regardless, everything should be sentence case. If this is an important feature, I should probably do a UI text review of it at some point.